src/inc/gcinfodecoder.h

```diff
@@ -138,10 +138,12 @@ inline BOOL IS_ALIGNED( void* val, size_t alignment )
 
 #endif
 
+#include "unreachable.h"
+
 // Stuff from check.h:
```

Comment:
The comment is now invalid.

Reply:
And actually, is this definition needed at all? It seems that neither this header nor the gcinfodecoder.cpp uses it, so it seems it doesn't belong here.

Reply:
I can just delete it if you don't mind.

Reply:
@janvorli oops, didn't see you suggested to remove it :)